service/src/main/java/com/ge/predix/acs/zone/management/dao/AttributeConnectorEntity.java

```diff
@@ -0,0 +1,50 @@
+package com.ge.predix.acs.zone.management.dao;
+
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "attribute_connector")
+public class AttributeConnectorEntity {
+    @Id
+    @Column(name = "id")
```

Comment:
Is is possible to move annotations on getters which I believe is a better practice read stackoverflow or google for more details?